packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -2752,16 +2740,7 @@ if (ROUTER_EVENTS) {
     },
   };
 
-  Route.reopen(ROUTER_EVENT_DEPRECATIONS, {
-    _paramsFor(routeName: string, params: {}) {
```

Comment:
Since this was in the `ROUTER_EVENTS` conditional it actually would have broken things in the future.